pkg/cloud/cloud.go

```diff
@@ -84,12 +88,17 @@ func (c *Client) Create(path, release, channel, arch string, version int) (err e
 	return
 }
 
-func extractVersionFromList(list, release, channel, arch string) (ver int, err error) {
+func (c *Client) extractVersionsFromList(release, channel, arch string) ([]string, error) {
```

Comment:
It would be nice to mention somehow that the returned list is ordered.
I would put it in the name extractOrderedVersionsFromList, but using long names is pythonic. And anyway it doesn't say which order. So, a comment for the func might be a good solution.